README.md

```diff
@@ -21,6 +21,17 @@ If you want to override the number of nodes,
 kurtosis run github.com/kurtosis-tech/cassandra-package '{"num_nodes": <required_number_of_nodes>}'
 ```
 
+If you want to run this with monitoring on top; use
+
+```
+kurtosis run github.com/kurtosis-tech/cassandra-package '{"monitoring_enabled": true}'
+```
+
+This will spin up Prometheus and Grafana on top of Cassandra using a `jmx_exporter`. If you copy the Grafana URL from the output of
```

Comment:
```suggestion
This will spin up local instances of Prometheus and Grafana connected to your Cassandra cluster using a `jmx_exporter`. If you copy the Grafana URL from the output of
```